meinberlin/assets/scss/components_user_facing/_moderator_status.scss

```diff
@@ -12,4 +12,20 @@
     width: 0.8em;
     height: 0.8em;
     border-radius: 50%;
-}
+
+    &.label--consideration {
+        background-color: $feedback-color-consideration;
+    }
+
+    &.label--qualified {
+        background-color: $feedback-color-consideration;
+    }
+
+    &.label--rejected {
+        background-color: $feedback-color-rejected;
+    }
+
+    &.label--accepted {
+        background-color: $feedback-color-accepted;
+    }
```

Comment:
these styles are currently defined in label.scss https://github.com/liqd/a4-meinberlin/blob/b1033aba0f8442c10fab3fa31c0e0deed6d380d0/meinberlin/assets/scss/components/_label.scss#L39
I think they should either be updated only there or deleted